src/dmd/backend/varstats.di

```diff
@@ -94,9 +96,12 @@ else
     /* Free function version:
      */
 
+    alias varWriter  = nothrow void function(Symbol*);
+    alias voidFunc   = nothrow void function();
+    alias beginBlock = nothrow void function(int off,int len);
```

Comment:
The D convention is to start types with a capital letter.

Reply:
Yeah and since these declarations will be modified, may I suggest to put `nothrow` at the tail ?